app/qml/Status/Application/MainView/StatusApplicationSections.qml

```diff
@@ -0,0 +1,25 @@
+import QtQml
+
+import Status.Controls.Navigation
+
+QtObject {
```

Comment:
The idea of this object is great. This gives uniformity and everything related to sections is concentrated in the same point.